docs/index.rst

```diff
@@ -0,0 +1,51 @@
+.. Fedimg documentation master file, created by
+   sphinx-quickstart on Sat May  5 00:26:21 2018.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+======
+Fedimg
+======
+
+Fedimg is a `Python`_-powered service that is built
+to make Fedora Cloud and Atomic Host images available on popular cloud
+providers, such as `Amazon Web Services`_.
+
+Salient features:
+
+* Performs automatic uploads, triggered by activity on the `Fedmsg`_ bus.
+* Register images as Hardware Virtual Machine (HVM).
+* Emits fedmsg messages, providing the the upload status and the Amazon Machine
+  Image (AMI) info.
+* Packaged for EPEL7 as well as PyPI.
```

Comment:
We may also want to mention Fedora? (latest build 1.3.0 is failing but this should get fixed)